docs/code-reviews/inclusion-in-code-review.md

```diff
@@ -0,0 +1,59 @@
+# Inclusion in Code Review
+
+You may be thinking why care about inclusiveness in code reviews. Below are some points which emphasize why inclusivity in code reviews is important:
+
+* Doing Code Review is an important part of our job as Software Professionals;
+* As CSE, we work with cross cultural teams from across the globe;
+* How we communicate while doing code review affects team morale;
+* Inclusive code reviews make new developer comfortable with the team;
+* Rude or personal attacks doing code reviews are very common, people unknowingly make rude comments in reviewing Pull Requests.
+
+## Types of non-inclusive code review behavior
+
+* Inequitable review assignment;
+* Negative Inter-Personal Interactions;
+* Biased Decision Making;
+
+## Examples of Non-Inclusive Code Reviews
+
+* Assigning lot of PRs to a single person for review and not assigning any PRs for other people in team. - _Inequitable review assignment_
+* Long arguments in the pull requests over subjective topics like style of coding . - _Negative Inter-Personal Interactions_
+* Commenting on the developer and not the code. Assuming code from developer X will always be good and hence not reviewing it properly and vice versa. - _Biased Decision Making_
+
+## Examples of Inclusive Code Reviews
+
+* Anyone and everyone in the team should be assigned PRs to review.
+* Reviewer should be clear about what is an opinion, their personal preference, best practice or a fact. Arguments over personal preferences and opinions are mostly avoidable.
+* Using inclusive language and tone in the code review comments. For example, being suggestive rather being prescriptive in the review comments is a good way to get the point across the table.
+* For the author of PRs- It's a good practice to thank the reviewer for the review if they have contributed in improving your code or you have learnt something new.
+* Using the Sandwich technique, for recommending a code change to a new developer or a new customer, sandwich the suggestion between 2 compliments. Something like- "Great Work so far, but i would recommend a few changes here. Btw, i loved the use of XYZ here, great work!!."
+
+## Responsibility of the author to make code reviews inclusive
+
+* Aim to write a code that is easy to read, review and maintain.
+* It’s important to ensure that whoever is looking at the code, whether that be the reviewer or a future engineer, can understand the motivations and how your code achieves that.
+* Proactively asking for targeted help or feedback.
+* Respond clearly to questions asked by the reviewers.
+* Avoid huge commits to submitting incremental changes.

```

Comment:
"Avoid huge commits to submitting incremental changes." - I'm not sure I understand exactly what is meant by this bullet point. Is it that one should favor small incremental changes over overhauls? I think this goes more into the pull request category and furthermore, how user stories are divided into tasks and their acceptance criteria etc. etc.

I suggest removing this line or rephrasing it to reveal the inclusiveness aspect of it.